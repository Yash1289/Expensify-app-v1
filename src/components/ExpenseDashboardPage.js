const React = require("react");
import ConnectedExpensesList from  "./ExpensesList"
import ExpensesListFilter from  "./ExpensesListFilter"
import ExpensesSummary from "./ExpenseSummary"

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummary />
        <ExpensesListFilter />
        <ConnectedExpensesList />
    </div>
)

export default ExpenseDashboardPage ;