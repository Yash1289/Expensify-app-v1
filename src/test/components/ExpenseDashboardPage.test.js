import { TestScheduler } from "jest"
import React from "react"
import { shallow } from "enzyme"
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage"


 test("Should render header component ", () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot(); 
}) 