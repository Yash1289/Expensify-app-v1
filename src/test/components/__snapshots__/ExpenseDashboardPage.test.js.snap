// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header component  1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
